### ISSUE TITLE
SPIKE: Find out what migration incentives are most persuasive for veterans

### ISSUE THREAD

**Issue by claytonzook** [department-of-veterans-affairs/va.gov-team#59027]
### SPIKE: Find out what migration incentives are most persuasive for veterans

Which veteran segments will be most persuaded by which features of Login.gov or ID.me? 

#### Relevant Docs
- [Mural board for brainstorm](https://app.mural.co/t/oddballinternal7486/m/oddballinternal7486/1685635348084/fe7c4f19f72a2fdc62c2ca4b9993158a76dfddb5?sender=u6c46f747a0483ea836054483)

#### Acceptance Criteria
- [ ] Reviewed by UX Team
- [ ] Reviewed by PMs
- [ ] Documentation has been updated
- [ ] Put published research in drive

## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by APorter-Clarity** [comment]
Pre-Discovery work (Brainstorming ideas for how this might work) Tentative Wednesday 5/31

**Comment by claytonzook** [comment]
Putting on hold for now while our teams address work splits, since this may not be part of our team's body of work in the near future.

**blocked by leadership decisions**

**Comment by aparsley** [comment]
This is a LGA initiative.
